### DISCUSSION TITLE
Failing to sign in to GitHub from Linux VSCode (maybe URI issue?)

### DISCUSSION THREAD

**Discussion by CABeteta** [community/community#14558] [Copilot Conversations]
I am having some issues loging into GitHub from a Linux VSCode. I might not be the only one...

 When I open the program I get the pop-up asking to log in, then I do it (using Firefox), but then when I have logged in this is what I get:
![Screenshot 2022-04-08 143825](https://user-images.githubusercontent.com/98741519/162438018-340be8f4-f712-41b0-8f34-eef6365f08e5.png)
 Seems like VSCode isn't opening the URI as it should.
 Any ideas?

Version information: 

VSCode Version: 1.66.1
Commit: 8dfae7a5cd50421d10cd99cb873990460525a898
Date: 2022-04-06T14:51:26.173Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Linux x64 5.13.0-35-generic snap
![Screenshot at 2022-04-08 14-47-05](https://user-images.githubusercontent.com/98741519/162438436-e5a579c2-956a-4e4d-b133-ed9c96b4f86e.png)

**Comment by ghost** [comment]
i have the same issue
